ob-dart.el

```diff
@@ -1,37 +1,33 @@
-;;; ob-dart.el --- org-babel functions for Dart evaluation
+;;; ob-dart.el --- Evaluate Dart source blocks in org-mode -*- lexical-binding: t; -*-
 
 ;; Copyright (C) 2016 Free Software Foundation, Inc.
 
 ;; Author: Milan Zimmermann
-;; Keywords: literate programming, reproducible research, emacs, org, babel, dart
+;; Maintainer: Milan Zimmermann
+;; Created: July 7, 2016
+;; Modified: May 19, 2022
+;; Version: 1.0.1
+;; Keywords: languages
 ;; Homepage: http://github.org/mzimmerm/ob-dart
-
-;; This is free software: you can redistribute it and/or modify
-;; it under the terms of the GNU General Public License as published by
-;; the Free Software Foundation, either version 3 of the License, or
-;; (at your option) any later version.
-
-;; This is distributed in the hope that it will be useful,
-;; but WITHOUT ANY WARRANTY; without even the implied warranty of
-;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-;; GNU General Public License for more details.
-
-;; For the GNU General Public License, see <http://www.gnu.org/licenses/>.
-
+;; Package-Requires: ((emacs "24.4"))
+;; SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Apparently it's enough to just name the LICENSE (hence I removed the GPL preamble below).